src/plugins/dashboard/public/application/utils/get_no_items_message.tsx

```diff
@@ -45,22 +45,16 @@ export const getNoItemsMessage = (
           <p>
             <FormattedMessage
               id="dashboard.listing.createNewDashboard.combineDataViewFromOpenSearchDashboardsAppDescription"
-              defaultMessage="You can combine data views from any OpenSearch Dashboards app into one dashboard and see everything in one place."
+              defaultMessage="You can combine data views from any Wazuh Dashboards app into one dashboard and see everything in one place."
```

Comment:
`nitpick`: replace `Dashboards` by `dashboard` because the application is called `Wazuh dashboard`

Reply:
Fix in commit [73fa093](https://github.com/wazuh/wazuh-dashboard/pull/311/commits/73fa093b1d043ecf48838dce2272123aa80123bf)